docs/docs/quickstart-inference.md

```diff
@@ -0,0 +1,283 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Get started with W&B Inference
+
+This guide shows you how to use W&B Weave with the W&B Inference service. You can build and trace LLM applications without setting up your own infrastructure or managing API keys from multiple providers.
+
+The W&B Inference service helps you learn Weave faster because:
+- You can start immediately without obtaining API keys from other providers
+- You get access to multiple models in one place, making it easier to compare their behavior
+- Your usage is automatically tracked, so you can see Weave's tracing features in action
+- You work with the same models and APIs you might use in production
+
+## What you'll learn
+
+In this guide, you'll:
+- Set up Weave and the W&B Inference service
+- Build a simple LLM application with automatic tracing
+- Compare multiple models
+- Evaluate model performance on a dataset
+- View your results in the Weave UI
+
+## Prerequisites
+
+Before you begin, you need:
+- A W&B account ([sign up free](https://app.wandb.ai/login?signup=true))
+- Python 3.8 or later
+- Basic Python knowledge
+
+## Step 1: Install Weave
+
+Install the required libraries:
+
+```bash
+pip install weave openai
+```
+
+Get your W&B API key from [https://wandb.ai/authorize](https://wandb.ai/authorize).
+
+## Step 2: Make your first traced LLM call
+
+Start with a simple example that uses Llama 3.1 8B through W&B Inference:
+
+```python
+import weave
+import openai
+
+# Initialize Weave - replace with your-team/your-project
+weave.init("my-first-weave-project")
+
+# Create an OpenAI-compatible client pointing to W&B Inference
+client = openai.OpenAI(
+    base_url='https://api.inference.wandb.ai/v1',
+    api_key="YOUR_WANDB_API_KEY",  # Replace with your actual API key
+    project="my-first-weave-project",  # Required for usage tracking
+)
+
+# Decorate your function to enable tracing
+@weave.op()
+def ask_llama(question: str) -> str:
+    response = client.chat.completions.create(
+        model="meta-llama/Llama-3.1-8B-Instruct",
+        messages=[
+            {"role": "system", "content": "You are a helpful assistant."},
+            {"role": "user", "content": question}
+        ],
+    )
+    return response.choices[0].message.content
+
+# Call your function - Weave automatically traces everything
+result = ask_llama("What are the benefits of using W&B Weave for LLM development?")
+print(result)
+```
+
+When you run this code, Weave:
```

Comment:
Move this to the top and/or add a sentence before the code sample that explains at a high level what's happening?

Reply:
Good call, done